src/site/stages/build/drupal/api.js

```diff
@@ -8,6 +8,13 @@ const { queries, getQuery } = require('./queries');
 
 const syswidecas = require('syswide-cas');
 
+const {
+  contentDir,
+  readAllNodeNames,
+  readEntity,
+} = require('../process-cms-exports/helpers');
+const assembleEntityTree = require('../process-cms-exports')(contentDir);
```

Comment:
:thinking: I think this should be
```suggestion
const entityTreeFactory = require('../process-cms-exports');
```
for now, and get the actually do the
```js
const assembleEntityTree = entityTreeFactory(buildOptions['cms-export-dir') || contentDir);
```
when we have access to `buildOption`. Otherwise, we'll read all the nodes initially from `buildOptions['cms-export-dir']`, but the descendant entities will be read from `contentDir`.